### ISSUE TITLE
OSMF - playback continues after stop

### ISSUE THREAD

**Issue by kfeinUI** [mangui/flashls#333]
1) Load up a live HLS stream.
2) Once playing, stop playback.

The output goes black for a moment, but then playback resumes. It appears to play through the remainder of the buffer. 

3) Hit play to restart playback.

It fails to play.

**Comment by mangui** [comment]
@kfeinUI I commited some changes today that might affect this.
https://github.com/mangui/flashls/commit/f59d9420f644b885798026fb8135866d8c80c84f
could you recheck with latest flashls/dev.
if still repro, plz share debug logs 
thanks,
Cheers,
Mangui

**Comment by kfeinUI** [comment]
Thanks! Fixes the stopping issue.

What remains is that when you play again after having stopped. It appears to play off the remaining buffer. The expected behavior for a live stream is for it to do a fresh load so that you may watch the live playback.

**Comment by mangui** [comment]
@kfeinUI 
not clear about how you trigger stop ? through API means ? there is no stop button on GrindPlayer

**Comment by kfeinUI** [comment]
Yes, through the API. Never mentioned using Grind. I'm using stock Strobe.

For live streams, we stop rather than pause.

**Comment by mangui** [comment]
ok. in fact the bug is that upon calling stop() the internal buffer is not flushed.
in theory you should not be able to playback the previously cached content after stop().
if you want to restart your live stream playback you should reload the whole stream.
